CHANGELOG.md

```diff
@@ -50,7 +52,7 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
 - When normalizing author names, complete enclosing braces are kept. [#10031](https://github.com/JabRef/jabref/issues/10031)
 - We enhanced the dialog for adding new fields in the content selector with a selection box containing a list of standard fields. [#10912](https://github.com/JabRef/jabref/pull/10912)
 - We store the citation relations in an LRU cache to avoid bloating the memory and out-of-memory exceptions. [#10958](https://github.com/JabRef/jabref/issues/10958)
-- Keywords filed are now displayed as tags. [#10910](https://github.com/JabRef/jabref/pull/10910)
+- Keywords field are now displayed as tags. [#10910](https://github.com/JabRef/jabref/pull/10910)
```

Comment:
Small comment: In future, try to separate these small side changes to other pull requests. Reason: If a PR takes long to review, these kind of changes will be "outdated" (e.g., causing conflicts, because other PRs also touch these places).

On Linux (and Mac) you can try [GitButler](https://gitbutler.com/) for "carving out" unrelated changes.

Reply:
> On Linux (and Mac) you can try [GitButler](https://gitbutler.com/) for "carving out" unrelated changes. 

GitButler is now supported on [Windows](https://blog.gitbutler.com/gitbutler-for-windows/), but for some reason it didn't work with JabRef 😅